src/app/views/users/UsersController.jsx

```diff
@@ -124,7 +126,7 @@ export class UsersController extends Component {
             }
             notifications.add(notification);
             console.error("Error mapping users to state: ", error);
-            log.event(`Error mapping users to state:\n${JSON.stringify(error)}`, log.error(error));
+            log.event(`Error mapping users to state`, log.error(error));
```

Comment:
Is there a reason we are using template literals for all of these log event messages?

Not sure if there is any difference in performance in JS between the interpolated strings and string literals, but I assume there would be as there is in most languages?  Just a thought, may be completely wrong so feel free to ignore me.

Reply:
Fixed